source/docs/configuration.md

```diff
@@ -65,7 +65,13 @@ Setting | Description | Default
 `post_asset_folder` | Enable the [Asset Folder](asset-folders.html)? | `false`
 `relative_link` | Make links relative to the root folder? | `false`
 `future` | Display future posts? | `true`
-`highlight` | Code block settings |
+`highlight` | Code block syntax highlight settings, by default this feature is powered by the highlight.js library |
```

Comment:
This setting only applies to highlight.js, so it's redundant to say "by default...".
I would say, `Code block syntax highlight settings, powered by [highlight.js](https://highlightjs.org/)`